### PR TITLE
updated index, toc, and Related Info

### PR DESCRIPTION
**Pull request by randyh62**
I moved the LICENSE.md file into the docs folder, added it to the end of the TOC, and changed the "Contributing to the Docs" on the top-level index to "Related information"

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -26,16 +26,8 @@ Our documentation is structured as follows:
...
-
-* `Document structure <https://rocm.docs.amd.com/en/develop/contribute/index.html#supported-formats>`_
-* `Toolchains <https://rocm.docs.amd.com/en/develop/contribute/toolchain.html>`_
-* `Documentation builds <https://rocm.docs.amd.com/en/develop/contribute/building.html>`_
-* `How to provide feedback <https://rocm.docs.amd.com/en/develop/contribute/feedback.html>`_
-
-Licensing information for all ROCm components is listed on our
-`Licensing <https://rocm.docs.amd.com/en/latest/about/license.md>`_ page.
+* You can find instructions to contribute to the documentation at `Contributing to ROCm  <https://rocm.docs.amd.com/en/latest/contribute/index.md>`_ section.
+* Licensing information for all ROCm components can be found at `Licensing <https://rocm.docs.amd.com/en/latest/about/license.md>`_ page.
```
Comment (LisaDelaney) [inline docs/index.rst]: "found at licensing page" feels awkward...and passive. Can we revert this section to how it was?

Comment (LisaDelaney) [reply docs/index.rst]: (if you don't like the bullets, it's fine to remove those, but I think the content was more clear/concise as it was)

File: docs/index.rst [hunk 2]
```diff
@@ -26,16 +26,8 @@ Our documentation is structured as follows:
...
 
-We welcome collaboration! If you'd like to contribute to our documentation, you can find instructions
-in our `Contributing to ROCm  <https://rocm.docs.amd.com/en/latest/contribute/index.md>`_ section, which discusses:
-
-* `Document structure <https://rocm.docs.amd.com/en/develop/contribute/index.html#supported-formats>`_
-* `Toolchains <https://rocm.docs.amd.com/en/develop/contribute/toolchain.html>`_
-* `Documentation builds <https://rocm.docs.amd.com/en/develop/contribute/building.html>`_
-* `How to provide feedback <https://rocm.docs.amd.com/en/develop/contribute/feedback.html>`_
-
-Licensing information for all ROCm components is listed on our
```
Comment (LisaDelaney) [inline docs/index.rst:40]: If you don't like licensing info lumped in with the contribution stuff, we can add a new heading.

Comment (randyh62) [reply docs/index.rst:40]: Hi Lisa, I updated this as we discussed. I removed the header, and removed the bullets, and restructured the sentences.

File: docs/index.rst [hunk 3]
```diff
@@ -26,16 +26,7 @@ Our documentation is structured as follows:
...
+To contribute to the documentation refer to `Contributing to ROCm  <https://rocm.docs.amd.com/en/latest/contribute/index.md>`_.
 
-* `Document structure <https://rocm.docs.amd.com/en/develop/contribute/index.html#supported-formats>`_
-* `Toolchains <https://rocm.docs.amd.com/en/develop/contribute/toolchain.html>`_
-* `Documentation builds <https://rocm.docs.amd.com/en/develop/contribute/building.html>`_
-* `How to provide feedback <https://rocm.docs.amd.com/en/develop/contribute/feedback.html>`_
-
-Licensing information for all ROCm components is listed on our
-`Licensing <https://rocm.docs.amd.com/en/latest/about/license.md>`_ page.
+Licensing information can be found at `Licensing <https://rocm.docs.amd.com/en/latest/about/license.md>`_.
```
Comment (LisaDelaney) [inline docs/index.rst:32]: This is passive voice...maybe something like "For licensing information, see..." or "You can find licensing information [here]."

**Review by vin-huang — APPROVED** [review]

**Pull request merged**